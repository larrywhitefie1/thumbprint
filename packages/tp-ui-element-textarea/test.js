const sass = require('node-sass');

it('compiles correctly', () => {
    const css = sass
        .renderSync({ file: 'packages/tp-ui-element-textarea/_index.scss' })
        .css.toString();
    expect(css).toMatchSnapshot();
});
