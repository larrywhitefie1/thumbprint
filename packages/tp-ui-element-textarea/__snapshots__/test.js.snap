// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles correctly 1`] = `
".tp-textarea {
  width: 100%;
  border: 1px solid #d3d4d5;
  border-radius: 4px;
  font-size: 16px;
  line-height: 24px;
  padding: 10px 14px;
  vertical-align: middle;
  resize: vertical;
  height: 152px;
  min-height: 104px; }
  .tp-textarea:focus {
    border-color: #009fd9; }
  .tp-textarea--bad-news {
    border-color: #ff5a5f;
    color: #ff5a5f; }
  .tp-textarea[disabled], .tp-textarea--disabled {
    background-color: #fafafa;
    color: #d3d4d5; }
"
`;
